client/reducers/initialState.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  auth: {
+    authenticated: false,
+    error: ''
+  },
+  common: {
+    appLoaded: false,
+    appName: 'calltocode'
+  },
+  projects: {
+    fetching: false,
+    projects: []
+  },
+  users: {}
+}
```

Comment:
I think this idea is interesting I just think that it becomes less clear what a reducer does when you look at just the reducer itself and don't see it's initial state

Reply:
That's true. But I saw many tutorials videos, and almost all of them advice the creation of something similar. This way it's also easier to avoid duplicated prop names. It's all centralized. All the reducers just pick a small portion of that initialState....

Reply:
Got it! Yeah that makes sense, let's see how it goes.
Good work!